config-user.default.php

```diff
@@ -53,13 +53,13 @@
 	'sort_order' => 'DESC',
 	'anon_access' => false,
 	'mark_when' => array (
-		'article' => true,
+		'article' => false,
```

Comment:
"when the article is viewed"

Reply:
I do not like this change

Reply:
I think true is probably a more expected default for this one, yeah.

Reply:
I agree

Reply:
Fixed